tests/random/draw.test-d.ts

```diff
@@ -0,0 +1,28 @@
+import * as _ from 'radashi'
+import { expectTypeOf } from 'vitest'
```

Comment:
Now that I see, this import is unnecessary. the other tests already use the vitest global functions
```suggestion

```